docs/source/asr/configs.rst

```diff
@@ -1082,7 +1088,16 @@ Fine-tuning via a Pytorch Lightning checkpoint
         trainer.devices=-1 \
         trainer.accelerator='gpu' \
         trainer.max_epochs=50 \
-        +init_from_ptl_ckpt="<name of pytorch lightning checkpoint>"
+        +init_from_nemo_model="<path to .nemo model file>" # (or +init_from_pretrained_model, +init_from_ptl_ckpt )
```

Comment:
Init from ptl ckpt is supported for this function, why is it being removed ?

Reply:
I just updated the doc to characterize finetuning based on tokenizer updating or model architecture updating. 
Hence condensed all the finetuning docs to one subsection, else it looks lengthy.